.github/workflows/test.yml

```diff
@@ -12,11 +12,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - run: npm ci
-
       - name: Boot up DynamoDB
         run: docker-compose up -d
 
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+
+      - run: npm install
```

Comment:
Why did you change `npm ci` to `npm install`?

Reply:
actually just playing around in order to check if it would help with the aws sdk error, it didn't so I'll rollback